lib/NewFangle/Agent.pm

```diff
@@ -3,10 +3,10 @@ package NewFangle::Agent;
 
 use strict;
 use warnings;
-use feature 'state';
 use experimental 'signatures';
 
 use Devel::Peek;
+use Class::Load 'load_optional_class';
```

Comment:
Ah, we stuck with Class::Load in the end?

Reply:
Yeah because there are three possibilities.

1. Module exists on disk and compiles (returns `1`)
2. Module exists on disk and doesn't compile (croaks)
3. Module doesn't exist on disk (returns `0`)

I couldn't get that functionality without it.

Reply:
`Module::Runtime` doesn't distinguish between cases 2 and 3.